bin/redshift/delete_aws_redshift.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```

Comment:
no need for `pwd`, see https://github.com/trinodb/trino/blob/dc03cff449540599e5d884281e7197cb73049956/plugin/trino-hive-hadoop2/bin/common.sh#L103-L105 for how we usually do stuff like this